docs/getting-started/clients/browser/index.md

```diff
@@ -8,6 +8,79 @@ sidebar_position: 3
 
 Before you start, you must complete the [Clients repository setup instructions](../index.md).
 
+## Environment Setup
+
+By default, the browser extension will run pointing to the production server endpoints. To override
+this for local development and testing, there are several options.
+
+### Using `managedEnvironment`
+
+The browser extension has the concept of a "managed environment", which is JSON configuration stored
+in
+[development.json](https://github.com/bitwarden/clients/blob/master/apps/browser/config/development.json).
+The `managedEnvironment` setting allows the contributor to override any or all of the URLs for the
+server. The `managedEnvironment` is read in the
+[`BrowserEnvironmentService`](https://github.com/bitwarden/clients/blob/master/apps/browser/src/services/browser-environment.service.ts)
+and overrides the default (production) settings for any supplied URLs.
+
+There are two ways to use `managedEnvironment`, depending upon whether you will also be running the
+web vault at the same time.
+
+#### `managedEnvironment` with web vault running
+
+If you are also running the web vault, you only need to set the `base` URL in the
+`managedEnvironment`:
+
+```json
+    "managedEnvironment": {
```

Comment:
I think including the full JSON structure provides better clarity. 
```json
{
   "devFlags":{
      "managedEnvironment":{
         "base":"https://localhost:8080"
      }
      ...
   }
   ...
}
```

Reply:
Good idea.  I've adjusted both cases.